tensorflow/python/autograph/operators/py_builtins.py

```diff
@@ -342,7 +342,24 @@ def _py_enumerate(s, start=0):
   return enumerate(s, start)
 
 
-SUPPORTED_BUILTINS = (abs, float, int, len, print, range, enumerate)
+def zip_(*s):
+  if all(isinstance(x, dataset_ops.DatasetV2) for x in s):
+    return _tf_dataset_zip(*s)
+  return _py_zip(*s)
+
+
+def _tf_dataset_zip(*s):
+  tup = ()
```

Comment:
This can be written simply as `tuple(s)` - can you given it a try?

Reply:
Ah, yea it works